src/index.js

```diff
@@ -158,21 +158,9 @@ function jumpToPast (history, index) {
 }
 // /jumpToPast
 
-// wrapState: for backwards compatibility to 0.4
-function wrapState (state) {
-  return {
-    ...state,
-    history: state
-  }
-}
-// /wrapState
-
 // updateState
 function updateState (state, history) {
```

Comment:
Is there any reason why we can't just return `res`? I think merging `state` with `res` was only needed with the `history`. The only problem I can see is when you want it to keep other state (besides `past`, `present` and `future`), which would get lost in this case. (although I'm not sure if this is something that even makes sense to do)